src/dashboard/src/main/migrations/0033_store_transfer_file_modification_dates.py

```diff
@@ -0,0 +1,112 @@
+    """
+
+    ###########################################################################
+    # Model Classes
+    ###########################################################################
+
+    TaskType = apps.get_model('main', 'TaskType')
+    StandardTaskConfig = apps.get_model('main', 'StandardTaskConfig')
+    TaskConfig = apps.get_model('main', 'TaskConfig')
+    MicroServiceChainLink = apps.get_model('main', 'MicroServiceChainLink')
+    MicroServiceChainLinkExitCode = apps.get_model('main', 'MicroServiceChainLinkExitCode')
+
+    ###########################################################################
+    # Get Existing Model Instances
+    ###########################################################################
+
+    one_instance_type = TaskType.objects.get(description='one instance')
+
+    ###########################################################################
+    # Store file modification dates CHAIN LINKs, etc.
+    ###########################################################################
+
+    # Store File Modification Dates Standard Task Config.
+    set_file_mod_dates_stc_pk = '102edcd9-3d0c-47d1-be1e-137875d73765'
+    StandardTaskConfig.objects.create(
+        id=set_file_mod_dates_stc_pk,
+        requires_output_lock=False,
+        execute='storeFileModificationDates_v0.0',
+        arguments='"%SIPUUID%" "%sharedPath%"'
+    )
+
+    # Store File Modification Dates Task Config.
+    set_file_mod_dates_tc_pk = 'fc64dc4c-e245-402b-b572-2dc027b414d1'
+    TaskConfig.objects.create(
+        id=set_file_mod_dates_tc_pk,
+        tasktype=one_instance_type,
+        tasktypepkreference=set_file_mod_dates_stc_pk,
+        description='Store file modification dates'
+    )
+
+    # Store File Modification Dates Chain Link.
+    set_transfer_file_mod_dates_cl_pk = 'f8ef02c4-f585-4b0d-9b6f-3cef6fbe527f'
+    fail_cl_pk = '61c316a6-0a50-4f65-8767-1f44b1eeb6dd'
+    MicroServiceChainLink.objects.create(
+        id=set_transfer_file_mod_dates_cl_pk,
+        currenttask_id=set_file_mod_dates_tc_pk,
+        microservicegroup='Characterize and extract metadata',
+        defaultnextchainlink_id=fail_cl_pk,
+        defaultexitmessage=Job.STATUS_FAILED
+    )
+
+    # Make "Policy checks for preservation derivatives" exit to the "Load
+    # Rights" chain link.
+    load_rights_cl_pk = '1a136608-ae7b-42b4-bf2f-de0e514cfd47'
+    MicroServiceChainLinkExitCode.objects.create(
+        id='4eaba00b-30bc-4508-84e4-d9871cd39fc7',
+        microservicechainlink_id=set_transfer_file_mod_dates_cl_pk,
+        nextmicroservicechainlink_id=load_rights_cl_pk,
+        exitcode=0,
+        exitmessage=Job.STATUS_COMPLETED_SUCCESSFULLY
+    )
+
+    # Introduce between "Add processed structMap to METS.xml document" and "Characterize and extract"
+    MicroServiceChainLinkExitCode.objects.filter(microservicechainlink_id='307edcde-ad10-401c-92c4-652917c993ed').update(nextmicroservicechainlink_id=set_transfer_file_mod_dates_cl_pk)
```

Comment:
Nitpick, if you're consistently using 79 columns in this file this is a clear violation of the rule! :P